core/src/test/scala/matryoshka/spec.scala

```diff
@@ -288,8 +291,8 @@ class MatryoshkaSpecs extends Specification with ScalaCheck with ScalazMatchers
       case _ => t.map(_.head).embed
     }
 
-  "Recursive" should {
-    "isLeaf" should {
+  "Recursive" >> {
+    "isLeaf" >> {
```

Comment:
Ah, well, I wouldn’t worry about it. These shouldn’t be nested anyway – the outer ones should be in their own specifications, but that’s a change for a different PR. At that point, maybe we can change the inners back, and rephrase things so they make more sense (which means better examples in a bunch of places, too).